packages/qemuqmp/default.nix

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  python3Packages,
+  fetchPypi,
+  lib,
+}:
+with python3Packages;
```

Comment:
no need for the with as you can be explicit in the line below. similar to the vhotplug/default.nix below

Reply:
Right. Fixed.